test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -607,5 +635,142 @@ public void SmokeInMemoryOnnxMapTypeTest()
                     Assert.Equal(pair.Value, transformedDataPoints[i].Output[pair.Key + 1]);
             }
         }
+
+        /// <summary>
+        /// A test to check if dynamic shape works.
+        /// </summary>
+        [OnnxFact]
+        public void TestOnnxTransformWithCustomShapes()
+        {
+            // The loaded model has input shape [-1, 3] and output shape [-1].
+            var modelFile = Path.Combine(Directory.GetCurrentDirectory(), "unknowndimensions", "test_unknowndimensions_float.onnx");
```

Comment:
>test_unknowndimensions_float [](start = 96, length = 28)

Could you add a link to the description of the onnx model this test is using?

Reply:
There is no such a description for those internal ONNX models but I can add a link to its source.

---
In reply to: [301726035](https://github.com/dotnet/machinelearning/pull/3963#discussion_r301726035) [](ancestors = 301726035)